src/main/java/com/ashkan/moviesapi/entities/ActorsInMovies.java

```diff
@@ -8,16 +8,18 @@
 @Entity
 @Table(name = "actors_in_movies", schema = "moviedb", catalog = "")
 public class ActorsInMovies {
+    @JsonIgnore
     private int actorsInMoviesId;
     private int actorId;
+    @JsonIgnore
     private int movieId;
     @JsonIgnore
     private Actor actorByActorId;
```

Comment:
This comment should be in line 67:
Since this is a relation **@ManyToOne**, I would like to recommend to use a fetch property, specifically **FetchType.LAZY**, please take a look what is the use of the fetch for this type of relations and if my suggestions is the recommended one or which one you need to use.
Please take this into account for all the relations between your Entity classes.

Reply:
done.